### DISCUSSION TITLE
Test discussion post for testing

### DISCUSSION THREAD

**Discussion by mcgratta** [firemodels/fds#11527] [General]
This is a test post

**Comment by rmcdermo** [comment]
Thanks.  Looks like this works.

**Comment by mcgratta** [comment]
I assume you mean responding to the Discussions directly with email? I'll
try it too.

**Comment by emanuelegissi** [comment]
It works indeed. Time to start thinking about doing the same for BlenderFDS
and qgis2fds.


Il sab 11 mar 2023, 17:34 Kevin McGrattan ***@***.***> ha
scritto:
